data/assets/educational/scale/gateway_arch.asset

```diff
@@ -58,29 +61,29 @@ local updatePositionAction = {
   IsLocal = false
 }
 
-local resetPositionAction = {
-  Identifier = "os.os.scale_models.reset_gateway_arch",
+local ResetPositionAction = {
+  Identifier = "os.scalemodels.ResetGatewayArch",
   Name = "Reset Gateway Arch position",
   Command = [[
-    openspace.globebrowsing.setNodePosition('Scale_GatewayArch', ']] .. earthAsset.Earth.Identifier .. [[', 38.624880, -90.184939)
-  ]],
-  Documentation = "Resets the Gateway Arch back to it's actual location",
+    openspace.globebrowsing.setNodePosition('Scale_GatewayArch', ']] .. earthAsset.Earth.Identifier .. [[', ]] .. Location[1] .. "," .. Location[2] .. ")",
+  Documentation = "Resets the Gateway Arch back to its actual location",
   GuiPath = "/Scale Objects",
   IsLocal = false
 }
 
 asset.onInitialize(function()
-  openspace.addSceneGraphNode(scaleModel)
-  openspace.action.registerAction(updatePositionAction)
-  openspace.action.registerAction(resetPositionAction)
+  openspace.addSceneGraphNode(ScaleModel)
+
+  openspace.action.registerAction(UpdatePositionAction)
+  openspace.action.registerAction(ResetPositionAction)
 end)
 
 asset.onDeinitialize(function()
-  openspace.action.removeAction(resetPositionAction)
-  openspace.action.removeAction(updatePositionAction)
-  openspace.removeSceneGraphNode(scaleModel)
+  openspace.action.removeAction(ResetPositionAction)
+  openspace.action.removeAction(UpdatePositionAction)
+  openspace.removeSceneGraphNode(ScaleModel)
```

Comment:
nitpicky, but in other cases you added an extra newline between actions and SGNs